nsql/README.md

```diff
@@ -0,0 +1,100 @@
+## Text to SQL (NSQL) Quickstarts
+
+Spice can provide an end to end system for users to ask questions of their SQL data. 
+
+## Requirements
+
+- [Spice CLI](https://docs.spiceai.org/getting-started) installed.
+- The following environment variables set:
+  - `SPICE_OPENAI_API_KEY`
+--- 
+
+**Step 1.** Run Spice runtime
+```shell 
+spice run
+```
+
+**Step 2.** Run the NSQL tool
+```shell
+spice nsql
+```
+
+**Step 3.** Ask a question of the dataset (`taxi_trips`).
+```shell
+>>> spice nsql
+Welcome to the Spice.ai NSQL REPL!
+Using model:
+ oai
+
+Enter a query in natural language.
+nsql> How many trips had more than two passengers?
++------------+
+| trip_count |
++------------+
+| 199155     |
++------------+
+
+Time: 1.006369 seconds. 1 rows.
+```
+
+**Step 4.** Check the underlying query
+```shell
+spice sql 
+```
+
+Then:
+```shell
+sql> SELECT start_time, parent_span_id, span_id, task, substr(input, 0, 64) AS input, execution_duration_ms FROM runtime.task_history WHERE trace_id=(SELECT trace_id FROM runtime.task_history WHERE task='nsql') ORDER BY start_time ASC;
```

Comment:
Create two separate fenced blocks. One containing the SQL to run using the `sql` formatter and another using the `shell` formatter with the results